app/views/shared/_filter_search.html.erb

```diff
@@ -1,6 +1,7 @@
 <script>
   var filterInput = document.getElementById('filter-input');
   filterInput.addEventListener('keyup', filterResults);
+  filterInput.addEventListener('search', filterResults);
```

Comment:
for browsers that support the `search` event, this will end up filtering the results twice.
while this won't technically cause an issue, it will result in the code running twice, resulting in noticeable slowdowns with larger lists

Reply:
Search only fires when you hit the x button.
Can confirm that they don't both fire when typing.